jsapp/js/components/processing/analysis/list/analysisQuestionRow.component.tsx

```diff
@@ -55,7 +57,20 @@ export default function AnalysisQuestionRow(props: AnalysisQuestionRowProps) {
     return null;
   }
 
-  const isDragDisabled = analysisQuestions.state.isPending;
+  // Reordering analysis questions requires `manage_asset` permission.
+  const hasManagePermissions = (() => {
+    const asset = assetStore.getAsset(singleProcessingStore.currentAssetUid);
+    return userCan('manage_asset', asset);
+  })();
```

Comment:
It looks like this IIFE gets duplicated in jsapp/js/components/processing/analysis/responseForms/commonHeader.component.tsx - might be good to import it instead. Or pull it out into a custom hook.